src/_layouts/default.html

```diff
@@ -1,12 +1,17 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    {% include meta.html %} {% include styles.html %} {%include analytics.html %}
+    {% include meta.html %} {% include styles.html %} {% include analytics.html %}
   </head>
 
   <body>
     {% include header.html %}
-    <main class="container">{{content}}</main>
+    <main class="container">{{ content }}</main>
     {% include footer.html %}
+
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/js/bootstrap.bundle.min.js"

```

Comment:
Version-matching to the Bootstrap CSS

```suggestion
      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
```